kafka/record/__init__.py

```diff
@@ -1,3 +1,3 @@
-from kafka.record.memory_records import MemoryRecords
+from kafka.record.memory_records import MemoryRecords, MemoryRecordsBuilder
 
-__all__ = ["MemoryRecords"]
+__all__ = ["MemoryRecords", "MemoryRecordsBuilder"]
```

Comment:
I'm not very familiar with this code, but I don't understand why this import is needed? Reading the bug description, it looks like the problems were all due to the code fixed in `kafka/record/memory_records.py` below... If the import were just forgotten, I would have thought it would have thrown problems before?

Reply:
No particular reason, just used in tests. We can remove it, yea

Reply:
I'm fine leaving it in if you think it should be there, I just didn't understand if there was something I was missing.